chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/Room.kt

```diff
@@ -32,6 +32,10 @@ data class Room(
         memberUserIds() += userId
     }
 
+    fun addAllUsers(userIds: Set<String>) {
```

Comment:
Is this function `internal` by default? Because it probably shouldn't ever be called by the user (and the same is true of `addUser` and `removeUser` above), because whatever effect they are expecting, it's probably not what they will get!

Reply:
If we had javadoc we could have communicated the consequences of this method easily 😆 I'm not sure how to make something accessible to the library (our RoomStore.kt) file but not more widely outside that - is this something you could help me with?

Reply:
I think you use the `internal` visibility level

Reply:
Done.